packages/eslint-plugin/src/rules/no-unsafe-argument.ts

```diff
@@ -162,114 +162,132 @@ export default createRule<[], MessageIds>({
     const services = getParserServices(context);
     const checker = services.program.getTypeChecker();
 
-    return {
-      'CallExpression, NewExpression'(
-        node: TSESTree.CallExpression | TSESTree.NewExpression,
-      ): void {
-        if (node.arguments.length === 0) {
-          return;
-        }
+    function checkUnsafeArguments(
+      args: TSESTree.Expression[] | TSESTree.CallExpressionArgument[],
+      callee: TSESTree.LeftHandSideExpression,
+      node:
+        | TSESTree.CallExpression
+        | TSESTree.NewExpression
+        | TSESTree.TaggedTemplateExpression,
+    ): void {
+      if (args.length === 0) {
+        return;
+      }
 
-        // ignore any-typed calls as these are caught by no-unsafe-call
-        if (isTypeAnyType(services.getTypeAtLocation(node.callee))) {
-          return;
-        }
+      // ignore any-typed calls as these are caught by no-unsafe-call
+      if (isTypeAnyType(services.getTypeAtLocation(callee))) {
+        return;
+      }
 
-        const tsNode = services.esTreeNodeToTSNodeMap.get(node);
-        const signature = FunctionSignature.create(checker, tsNode);
-        if (!signature) {
-          return;
-        }
+      const tsNode = services.esTreeNodeToTSNodeMap.get(node);
+      const signature = FunctionSignature.create(checker, tsNode);
+      // istanbul ignore next
+      if (!signature) {
+        return;
+      }
```

Comment:
[Testing] Same feedback here: either a `nullThrows` or a test case to exercise this please 😇

Reply:
@JoshuaKGoldberg I couldn't think of a test case that covered this line, so I used nullThrows